programs/mango-v4/src/health/account_retriever.rs

```diff
@@ -241,21 +275,22 @@ impl<'a, 'info> ScannedBanksAndOracles<'a, 'info> {
             (index2, index1, true)
         };
 
+        let (price1, price2) = {
+            let (first_bank_part, second_bank_part) = self.banks.split_at(first + 1);
+            let bank1 = first_bank_part[first].load_fully_unchecked::<Bank>()?;
+            let bank2 = second_bank_part[second - (first + 1)].load_fully_unchecked::<Bank>()?;
```

Comment:
Reading this again, I think it can be simplified to
```
            let bank1 = self.banks[first].load_fully_unchecked::<Bank>()?;
            let bank2 = self.banks[second].load_fully_unchecked::<Bank>()?;
```
because with immutable borrows we don't need to split the banks Vec into two parts.